packages/sdk-node/README.md

```diff
@@ -1,7 +1,256 @@
-# Looker SDK Library for Node
+# Looker SDK for Node
 
-You will need this package if you want to run the Looker SDK in a node environment. This package is NOT required if you want to the run the Looker SDK in a browser environment.
+The Looker SDK for Typescript/Javascript works with Node and browser run-times. The SDK provides a convenient way to communicate with a Looker server's APIs.
 
-The `@looker/sdk` package is updated with every Looker release. This package has a much longer update/release cycle.
+This package is specifically for using the Looker Typescript SDK with Node. It depends on the [@looker/sdk](https://www.npmjs.com/package/@looker/sdk) package and [@looker/sdk-rtl](https://www.npmjs.com/package/@looker/sdk-rtl).
 
-**DISCLAIMER**: This is a _beta_ version of the Looker SDK library for Node. Implementations are still subject to change.
+The SDK uses a plug-in architecture (also known as dependency injection) for initializing that supports run-time specific transports (like `NodeTransport` and `BrowserTransport`) and different approaches for managing API authentication (like `NodeSession`, `BrowserSession`, `ProxySession`, and `OauthSession`).
+
+**DISCLAIMER**: This is a _beta_ version of the Looker SDK. Implementations are still subject to change, but SDK method calls are expected to work correctly. Please [report any issues](https://github.com/looker-open-source/sdk-codegen/issues) encountered, and indicate the SDK language in the report.
+
+## Getting started
+
+The Looker SDK can be used in a node application in 3 steps:
+
+- install
+- configure
+- use
+
+### Install the Looker SDK into your node application
+
+Using `yarn`:
+
+```bash
+yarn add @looker/sdk @looker/sdk-node
+```
+
+Using `npm`:
+
+```bash
+npm install @looker/sdk @looker/sdk-node
+```
+
+**Note**: If you are only intending to use the SDK in a browser, omit `@looker/sdk-node`.
```

Comment:
Is this going to cause confusion of the sdk and sdk-rtl are specified explicitly above? 

My feeling is that we should explicitly mention sdk and sdk-rtl. I know they are implicitly pulled in the linting rules for our repo indicate an error if an import references a package that is not listed in the dependencies.

Reply:
I made the sdk-rtl imports explicit